lib/features/cleanup/data/local/recent_search_cache_manager.dart

```diff
@@ -76,6 +88,10 @@ class RecentSearchCacheManager {
         ? listRecentSearch.sublist(0, newLimit)
         : listRecentSearch;
 
+    for (var recentSearch in newListRecentSearch) {
+      log('RecentSearchCacheManager::getAllLatest: Recent search: ${recentSearch.toString()}');
+    }
```

Comment:
```suggestion
    if (kDebugMode) {
      for (var recentSearch in newListRecentSearch) {
        log('RecentSearchCacheManager::getAllLatest: Recent search: ${recentSearch.toString()}');
      }
    }
```

IMO we should not make the app wait to log on release build.

Reply:
We will delete it.

Reply:
Done